packages/colorpickers/src/styled/ColorPicker/StyledAlphaField.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { Field } from '@zendeskgarden/react-forms';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'colorpickers.colorpicker.alpha_field';
+
+/**
+ * 1. IE11 specific CSS to achieve styling parity with modern browsers.
+ */
+export const StyledAlphaField = styled(Field).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  top: ${props => props.theme.space.base * 2}px;
+
+  input {
+    /* stylelint-disable-next-line declaration-no-important */
+    margin-top: 0 !important;
+    height: ${props => props.theme.space.base * 3}px;
+  }
+
+  /* stylelint-disable-next-line */
+  @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) {
+    input {
+      top: -10px; /* [1] */
+      height: ${props => props.theme.space.base * 8}px; /* [1] */
+    }
+  }
```

Comment:
Shouldn't these styles be directly on `StyledAlpha`. Also, it appears that `height` is an unnecessary override.

Reply:
I don't think the styles can be on `StyledAlpha` since they target the `input`. I was able to remove the `height` for non-IE browsers, but IE requires it so that the thumb shadow styles don't get cut off.

Reply:
I'm confused. Isn't `StyledAlpha` the `<input type="range">`?

Reply:
Oops, it is. I've moved over all the input styles to `StyledAlpha` and `StyledHue` accordingly 👍🏽 .